modules/access_affinitygroup/access_affinitygroup.module

```diff
@@ -852,6 +852,35 @@ function access_affinitygroup_user_login(UserInterface $account) {
   }
 }
 
+/**
+ * Implements hook_user_presave().
+ * If user's email address changes, tell simplelists email.
+ */
+function access_affinitygroup_user_presave($uu) {
+
+  if ($uu->original->getEmail() <> $uu->getEmail()) {
+    $msg = "";
+    $simpleListsApi = new SimpleListsApi();
+    \Drupal::logger('access_affinitygroup')->debug("User email changed: " . $uu->getEmail() . ' from ' . $uu->original->getEmail());
+    $status = $simpleListsApi->updateUserEmailAddress($uu->original->getEmail(), $uu->getEmail(), $msg);
+
+    // email not found in simplelists account.
+    if (!isset($status)) {
+      return;
+    }
+
+    // User is a contact in simplelists. Change email address.
```

Comment:
I think this comment should be the email address was changed?

Reply:
no, the comment means change the email address in the simplelists contact.